spec/factory_bot/user.rb

```diff
@@ -15,5 +15,12 @@
       email { 'admin.john.doe@codelabs.test' }
       roles { %i[confirmed moderator administrator] }
     end
+
+    trait :confirmed do
+      name { 'John Doe Confirmed' }
+      username { 'conf.John.Doe' }
+      email { 'conf.john.doe@codelabs.test' }
```

Comment:
Could we use sequence here too? Is there even a reason to change these values for a confirmed user?